src/component/Counter/Counter.jsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+
+export class Counter extends React.Component {
+  state = {
+    count: 0,
+  };
+
+  addOne = () => {
+    this.setState(({ count }) => ({ count: count + 1 }));
+  };
+
+  add100 = () => {
+    this.setState(({ count }) => ({ count: count + 100 }));
+  };
+
+  increase = () => {
+    this.setState(({ count }) => {
+      this.addOne();
+      if (count % 5 === 0) {
+        this.add100();
+      }
+    });
+  };
```

Comment:
```suggestion
  increase = () => {
    this.addOne();
    
    if (count % 5 === 0) {
      this.add100();
    }
  };
```
No need in setState here. You call two functions that already call `setState` method inside them